angular-portfolio/src/app/award-detail/award-detail.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Awards } from '../awardsTemplate';
+import { AwardsService } from '../awards.service';
+
+@Component({
+  selector: 'app-award-detail',
+  templateUrl: './award-detail.component.html',
+  styleUrls: ['./award-detail.component.scss']
+})
+export class AwardDetailComponent implements OnInit {
+
+  awards: Awards[];
```

Comment:
The relationship between "award" and "awards" is unclear. Why do we need to request the entire list of rewards when we've already used Input for the one we want?

Reply:
*Discussed during 1-on-1" Recap: I inputted a particular "award" and the entire list as I want to select all awards that match that award's year. However, I will revise so that the child just gets the award.year as an integer rather than an object for better memory saving.